src/metrics/metrics.jl

```diff
@@ -337,36 +316,32 @@ juvenile_indicator = Metric(_juvenile_indicator, (:timesteps, :sites, :scenarios
     coral_evenness(X::AbstractArray{T})::AbstractArray{T} where {T<:Real}
     coral_evenness(rs::ResultSet)::AbstractArray{T} where {T}
 
-Calculates evenness across functional coral groups in ADRIA.
+Calculates evenness across functional coral groups in ADRIA as a diversity metric.
 Inverse Simpsons diversity indicator.
 
-# Notes
-Number of taxa (distinct groups with enhanced lumped with unenhanced) is hardcoded in this function.
-
 # References
 1. Hill, M. O. (1973).
     Diversity and Evenness: A Unifying Notation and Its Consequences.
    Ecology, 54(2), 427-432.
    https://doi.org/10.2307/1934352
 
 """
-function _coral_evenness(X::AbstractArray{T})::AbstractArray{T} where {T<:Real}
-    x::AbstractArray{<:Real} = min.(max.(X, 0.0), 1.0)
-    covers::NamedTuple = coral_cover(x)
-
+function _coral_evenness(r_taxa_cover::AbstractArray{T})::Array{T} where {T<:Real}
     # Evenness as a functional diversity metric
-    n::Int64 = 4  # number of taxa
-    p1::AbstractArray{<:Real} = dropdims(sum(covers.tab_acr, dims=2), dims=2) ./ covers.relative_cover
-    p2::AbstractArray{<:Real} = dropdims(sum(covers.cor_acr, dims=2), dims=2) ./ covers.relative_cover
-    p3::AbstractArray{<:Real} = dropdims(sum(covers.small_enc, dims=2), dims=2) ./ covers.relative_cover
-    p4::AbstractArray{<:Real} = dropdims(sum(covers.large_mass, dims=2), dims=2) ./ covers.relative_cover
-
-    sum_psqr::AbstractArray{<:Real} = p1 .^ 2 + p2 .^ 2 + p3 .^ 2 + p4 .^ 2  # functional diversity
-    simpson_D::AbstractArray{<:Real} = 1 ./ sum_psqr  # Hill 1973, Ecology 54:427-432
-    return simpson_D ./ n  # Group evenness
+    n_steps, n_grps, n_locs = size(r_taxa_cover)
+
+    # Sum across groups represents functional diversity
+    # Group evenness (Hill 1973, Ecology 54:427-432)
+    loc_cover = dropdims(sum(r_taxa_cover, dims=2), dims=2)
+    simpson_D = zeros(n_steps, n_locs)
```

Comment:
Maybe a comment to say what simpson_D stands for - is is SImpson's diversity?

Reply:
It is Simpson's diversity indeed. This was older code but I've now adjusted